api/src/main/java/org/eclipse/microprofile/concurrent/ManagedExecutor.java

```diff
@@ -70,9 +70,17 @@
  * <p>This interface is intentionally kept compatible with ManagedExecutorService,
  * with the hope that its methods might one day be contributed to that specification.</p>
  *
- * <p>Managed executors are managed by the container, not be the user. Therefore, all
- * life cycle methods must raise IllegalStateException. This includes:
- * awaitTermination, isShutdown, isTerminated, shutdown, shutdownNow</p>
+ * <p>Managed executors that are created with the {@link Builder} or have a life cycle that is
+ * scoped at or beneath the level of the application must support the life cycle methods, including:
```

Comment:
> I also think that we should aim for same behaviour of MEs obtained via builder and injection.

It was actually my intent to make the CDI config and builder path identical here.  The distinction I'm trying to make is against the EE Concurrency requirement to reject the life cycle methods for their managed executor instances are the scoped to the whole server.  I didn't want to use the word server here because not all MicroProfile implementations are application servers.  But in doing so, it seems I've made it more confusing.  I'll redo this another time and try to get it right.